### PR TITLE
RANGER-5046: checkstyle compliance updates - plugin-ozone

### PR DESCRIPTION
**Pull request by kumaab**
## What changes were proposed in this pull request?

Updated `plugin-ozone` module sources to comply with the check-style requirements.

## How was this patch tested?
`mvn clean verify` on local machine.

### CODE REVIEW

File: plugin-ozone/src/main/java/org/apache/ranger/services/ozone/client/OzoneClient.java [hunk 1]
```diff
@@ -151,43 +181,12 @@ public List<String> getKeyList(String keyPrefix, List<String> finalVolumeList, L
         } catch (IOException e) {
             LOG.error("Unable to get Volume List");
             if (LOG.isDebugEnabled()) {
-                LOG.debug("<== OzoneClient.getVolumeList() Error : " , e);
+                LOG.debug("<== OzoneClient.getVolumeList() Error : ", e);
             }
         }
         if (LOG.isDebugEnabled()) {
             LOG.debug("<== OzoneClient.getVolumeList(): " + ret);
```
Comment (mneethiraj) [inline plugin-ozone/src/main/java/org/apache/ranger/services/ozone/client/OzoneClient.java]: Replace string concat with parameterized log call. Please review all other occurrences as well.

File: plugin-ozone/pom.xml [hunk 2]
```diff
@@ -28,6 +28,7 @@ limitations under the License.
     <name>Ozone Security Plugin</name>
     <description>Ozone Security Plugin</description>
     <properties>
+        <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
```
Comment (mneethiraj) [inline plugin-ozone/pom.xml:31]: `<checkstyle.skip>false</checkstyle.skip>` would be needed as well.

Comment (kumaab) [reply plugin-ozone/pom.xml:31]: yup. added it.

File: plugin-ozone/src/main/java/org/apache/ranger/authorization/ozone/authorizer/RangerOzoneAuthorizer.java [hunk 3]
```diff
@@ -39,178 +39,168 @@
...
+        }
+
+        if (rangerPlugin == null) {
+            MiscUtil.logErrorMessageByInterval(LOG, "Authorizer is still not initialized");
+            return false;
+        }
+
+        //TODO: If source type is S3 and resource is volume, then allow it by default
+        if (ozoneObj.getStoreType() == OzoneObj.StoreType.S3 && ozoneObj.getResourceType() == OzoneObj.ResourceType.VOLUME) {
+            if (LOG.isDebugEnabled()) {
```
Comment (mneethiraj) [inline plugin-ozone/src/main/java/org/apache/ranger/authorization/ozone/authorizer/RangerOzoneAuthorizer.java]: I suggest removing LOG.isDebugEnabled(). Please review and update other occurrences as well.

**Review by mneethiraj — APPROVED** [review]

**Review by mneethiraj — APPROVED** [review]

**Pull request merged**